Content-Type: text/html

<br/>
DEBUG: dataProcess(uploadedFiles/a69993d5dfe7d97b33076052ae1d0c2e,supplementaryMaterials-alignedOnly.afa)<br/>

<br/>
DEBUG: listing: ['cds_ARG_Mouse.fa', 'cds_ARG_Chimp.fa', 'cds_ARG_Rhesus.fa', 'cds_ARG_Human.fa', 'cds_ARG_Marmoset.fa']<br/>

<br/>
DEBUG: infile: cds_ARG_Mouse.fa<br/>

<br/>
DEBUG: CDSExonCount: 0, startFrame: ^, stopFrame: <, CDSlength: 85, exonLength: 85<br/>

<br/>
DEBUG: CDSExonCount: 1, startFrame: <, stopFrame: >, CDSlength: 50, exonLength: 52<br/>

<br/>
DEBUG: CDSExonCount: 2, startFrame: >, stopFrame: ^, CDSlength: 138, exonLength: 139<br/>

<br/>
DEBUG: CDSExonCount: 3, startFrame: ^, stopFrame: >, CDSlength: 212, exonLength: 212<br/>

<br/>
DEBUG: CDSExonCount: 4, startFrame: >, stopFrame: ^, CDSlength: 132, exonLength: 133<br/>

<br/>
DEBUG: CDSExonCount: 5, startFrame: ^, stopFrame: >, CDSlength: 98, exonLength: 98<br/>

<br/>
DEBUG: CDSExonCount: 6, startFrame: >, stopFrame: ^, CDSlength: 129, exonLength: 130<br/>

<br/>
DEBUG: CDSExonCount: 7, startFrame: ^, stopFrame: >, CDSlength: 215, exonLength: 215<br/>

<br/>
DEBUG: CDSExonCount: 8, startFrame: >, stopFrame: ^, CDSlength: 132, exonLength: 133<br/>

<br/>
DEBUG: CDSExonCount: 9, startFrame: ^, stopFrame: >, CDSlength: 98, exonLength: 98<br/>

<br/>
DEBUG: CDSExonCount: 10, startFrame: >, stopFrame: ^, CDSlength: 138, exonLength: 139<br/>

<br/>
DEBUG: CDSExonCount: 11, startFrame: ^, stopFrame: >, CDSlength: 227, exonLength: 227<br/>

<br/>
DEBUG: CDSExonCount: 12, startFrame: >, stopFrame: ^, CDSlength: 132, exonLength: 133<br/>

<br/>
DEBUG: CDSExonCount: 13, startFrame: ^, stopFrame: ^, CDSlength: 69, exonLength: 69<br/>

<br/>
DEBUG: len: 620, AA: MEVLKALPFIVFFSYSKCESLQTSLLHAARQHVSGQDHLEENLGVSVSIMVAQFLQDAAYDQVQTIAKELLKLAEKCKRLKPESPSECAHQLVAAFLIHICNNQGLVDSHVFTDCCKMKTAARLRCFLSYKKDDADSHDTPLIPRPELTCEVAAENNVSLKERFSYEISRRHPFLYGPTILSMSACYDTAVQSCCQEENETECLRTKLEPIRKYIRGISARHHHLCEIGTKFNEKVASAVELILLTKKQPKANFSEIAKLTMDVKNLHETCCEGNTLVCALGRRQLMNYTCSNQAILSSKFAECCEQPEPFRGECIVNSENDDKPDLSPLPLTRFTEDWSVCQQFSGNQDSFLQEFLYEYSRRHLELAVPVILRVFETYKRVLEKCCKLENSLECHQPGKEMFQKVILESRDQVKTYCDLQKQLGSSSFQDRLTVLYTKKAPQLSAQELVVFVKKMAAVARRCCPLRDELQSACVEDQTKLILGELCRRHGGKPVNAGVGHCCDDSYAFRKPCFDDLRVDGTYISPPLSCDQAISLKEDLCQAQDEELQMEKQKLLSTLVKQKPHAAEDVFHSIGENFVQLVEKCCGAQRRDVCFQEEGPQLITKCQSLLEASSSQSGLM<br/>

<br/>
DEBUG: infile: cds_ARG_Chimp.fa<br/>

<br/>
DEBUG: CDSExonCount: 0, startFrame: ^, stopFrame: <, CDSlength: 85, exonLength: 85<br/>

<br/>
DEBUG: CDSExonCount: 1, startFrame: <, stopFrame: <, CDSlength: 49, exonLength: 51<br/>

<br/>
DEBUG: CDSExonCount: 2, startFrame: <, stopFrame: <, CDSlength: 139, exonLength: 141<br/>

<br/>
DEBUG: CDSExonCount: 3, startFrame: <, stopFrame: <, CDSlength: 130, exonLength: 132<br/>

<br/>
DEBUG: CDSExonCount: 4, startFrame: <, stopFrame: >, CDSlength: 188, exonLength: 190<br/>

<br/>
DEBUG: CDSExonCount: 5, startFrame: >, stopFrame: ^, CDSlength: 210, exonLength: 211<br/>

<br/>
DEBUG: CDSExonCount: 6, startFrame: ^, stopFrame: <, CDSlength: 133, exonLength: 133<br/>

<br/>
DEBUG: CDSExonCount: 7, startFrame: <, stopFrame: <, CDSlength: 97, exonLength: 99<br/>

<br/>
DEBUG: CDSExonCount: 8, startFrame: <, stopFrame: >, CDSlength: 311, exonLength: 313<br/>

<br/>
DEBUG: CDSExonCount: 9, startFrame: >, stopFrame: <, CDSlength: 133, exonLength: 134<br/>

<br/>
DEBUG: CDSExonCount: 10, startFrame: <, stopFrame: <, CDSlength: 31, exonLength: 33<br/>

<br/>
DEBUG: Trimming 1 nucleotides from ATGAAGACTGCTAAACCCACATCTTTTCTTGTTTTCTTCAGCTACATTAAATGTCAATCTCGTTAGAGAAGTTTACTAGATGAGGCCATACAGTATATTAATAGCCAGAATTACCCAGACAGAATGTAAGAGACATCCCCAACATTATGGTTGCTCAGTTTCTTCAGAAAGCAACTCATGGAGAAGTACAAACCACAGTTAGAAAGTTGAGAGATTTCAATGAGAAGCACAGGACCTTTAAGTCACATGGTTACCTTCAGAATGCACTCACCAATTGGTACAGTTATGAAACATCTAGAAGTCATCCATTCTTGTATAGACTTACTCTCCTGACATGTCTACTTGCTATGAAACAGCTGTTTAGTCATGTTGCCAAGAAGAAAAGAAGACAGAATGCCTCTAGATAAAGTTAGAATGCAACAAAAATATATTAGAGAATTATCTTTGAGATATCATAATTTATGTGAAATTATTTTGAAAAGTGCTTGTTCATATCTTTTGCCCACTTTTTGATGGGGTTGTTTGTTTTTTCTTATATATTTGTATAAGTTTCTTATAGATGCTGTAATTAAATTTATTTATGAAGTAGAAAAAGCAGTAGCCAGTTTATAAACTACCCCTGCTCTAAACAGGCTATCTTATCCAGCAAGATCACTCAATCCTGTGAACAGCCAGTGCCATTCCAAGGGGAAGGCATATTAACTCAGAAAATGCTGACACACCTGACCTTTCATCCCTGCCACTCAGCAGTTCTACAGAAGACCAGTTTGTATGCAAACAATTCACTTACAAGCCAGAATTCCTACAAGCGTTTCCTTATGAATATTCAAAAAGATATCCAAAGTTGGCAGTTCCAGTGATTTTAAGAGTGGATATAGTATATAAAAATGGATTGAGAAAGTGCTACAAATTAGAAAACCCTTTGGAATGCTATAGCTGTGGGAAGCAGATTTTGCAAAGAGTGGTATGTAAAAAGCAATTATCATGTAAAAAATCATTATGATTTACATGAGAAGTTAGGAGGTAGTAACTTCCATGACAGGTTCATAGTCTTCTACAGTAAGAAAGCCCCACAACCATCTGCTCAAGAGTTGGTTGTAAAATTTGGCAGGCATTGTGATGCATGCCTCTAATCCCAGCTACTTGGGAGGCTGAGGCAGGAGAATTACTCGAACCTGGGAGGTGGAGGTTGCAGTGAGCCAAGATCGCGCCCCTGCACTCCAGCCTGGGCAACAGGGTGAGACTCCTGTCTCAAACAAATAGAGTAGATTGTGTTCACAAAGATGATGGTAGCTGCTGCCACCAAATGTTGCCCACTGAGACATAAGCAACAGTTTGTCTGCATTGAGAACTCGGCTTATGAGCAACCTTGTGAAGCAGGAACCATGTGCAACAGGGATGCAGTTCCAATCAATTTTTGTGGATTTGGTTCACCTGTTGGAGATGAGCTGCCAAGGAAAGAAAAGTGAAACTGTGTTTTCAAGAGACTGAAAGCAGGAAAGATCAAAAATTGACATCCTAA<br/>

<br/>
DEBUG: len: 483, AA: MKTAKPTSFLVFFSYIKCQSRRSLLDEAIQYINSQNYPDRMETSPTLWLLSFFRKQLMEKYKPQLESEISMRSTGPLSHMVTFRMHSPIGTVMKHLEVIHSCIDLLSHVYLLNSCLVMLPRRKEDRMPLDKVRMQQKYIRELSLRYHNLCEIILKSACSYLLPTFWGCLFFLIYLYKFLIDAVIKFIYEVEKAVASLTTPALNRLSYPARSLNPVNSQCHSKGKAYLRKCHTPFIPATQQFYRRPVCMQTIHLQARIPTSVSLIFKKISKVGSSSDFKSGYSIKWIEKVLQIRKPFGMLLWEADFAKSGMKAIIMKIIMIYMRSEVVTSMTGSSSTVRKPHNHLLKSWLNLAGIVMHASNPSYLGGGRRITRTWEVEVAVSQDRAPALQPGQQGETPVSNKSRLCSQRWLLPPNVAHDISNSLSALRTRLMSNLVKQEPCATGMQFQSIFVDLVHLLEMSCQGKKSETVFSRDKQERSKIDIL<br/>

<br/>
DEBUG: infile: cds_ARG_Rhesus.fa<br/>

<br/>
DEBUG: CDSExonCount: 0, startFrame: ^, stopFrame: <, CDSlength: 85, exonLength: 85<br/>

<br/>
DEBUG: CDSExonCount: 1, startFrame: <, stopFrame: <, CDSlength: 49, exonLength: 51<br/>

<br/>
DEBUG: CDSExonCount: 2, startFrame: <, stopFrame: <, CDSlength: 139, exonLength: 141<br/>

<br/>
DEBUG: CDSExonCount: 3, startFrame: <, stopFrame: ^, CDSlength: 129, exonLength: 131<br/>

<br/>
DEBUG: CDSExonCount: 4, startFrame: ^, stopFrame: >, CDSlength: 179, exonLength: 179<br/>

<br/>
DEBUG: CDSExonCount: 5, startFrame: >, stopFrame: >, CDSlength: 209, exonLength: 210<br/>

<br/>
DEBUG: CDSExonCount: 6, startFrame: >, stopFrame: ^, CDSlength: 132, exonLength: 133<br/>

<br/>
DEBUG: CDSExonCount: 7, startFrame: ^, stopFrame: <, CDSlength: 100, exonLength: 100<br/>

<br/>
DEBUG: CDSExonCount: 8, startFrame: <, stopFrame: ^, CDSlength: 327, exonLength: 329<br/>

<br/>
DEBUG: CDSExonCount: 9, startFrame: ^, stopFrame: >, CDSlength: 134, exonLength: 134<br/>

<br/>
DEBUG: Trimming 2 nucleotides from ATGAAGACTGTTAAATCCACATCTTTTCTGGTTTTCTTCAGCTACATTAAATGTCAATCTCTTTAGAGAAGTTTATTAGATGCGGCCATACAGTATATTAATAGCCAGAATTACCCAGAGAGAATGTAAGAGACATCCCCAACATTATGGTTGCTCATCTTCTTCAGAAAGCCACTCATGGAGAAGTACAAACCATAGTTAGAAAGTTGAGAGATTTCAATGAGAAGCACAGGACCTTTAAGTCACATGGTTACCTTCAGAATGCACTCACCAATTGGTACAGTTATGAAACATCTAGAAGTCATCCATTCTTTTATAGACTTCCTCTCCTGACATGTCTACTTGCTATGAAACAGCTGTTTAGTCATGTTGCCAAGAAGAAAAAAGACAGAGTGCTTCTAGATAAAGCTAGAATGCAACAAAAATACATTAGAGAATTACCTTTGAGATATCATAATTTATGTGAAATTATTTTGAAAAGTGTTTGTTCATGTCTTTTGCCCACTTTTTGATGGCTTTTTTTCTTATATATTTAAGTTTCTTATAGATGCTGTAATTAAATTTAATTATGAAGTAGAAAAAGCAGTAGCCAGCTTATGAACTACCCCTGCTCTAAACAGGCTATCCTATCCAGCAAGATCACTCAATCCTGTGAACAGCCAATGCCATTCCAAGGAGAAGGCATATTAACTCAGAAAATGCTGACACACCTGACCTTTCATCCCTGCCACTAGTAGTTCTACAGAAGACCAGTGTGCATGCAAACAATTCACTTACAAGCAAGAATTCCTACAAGTGTTTCCTTATGAATATTCAAAAAGATATCCAAAGTTGGCAGTTCCAGTGATTTTAAGAGTGGATATAGTATATAAAAATGGATTGACAAAGTGCTACAAGTTAGAAAACCCTTTGGAATGCTATAGCTGTGGGAAGGAGATTTTGCGAAGAGTGGTATGTAAAAAGCAATTATCATGTAAAATAATCATTACGATTTACATGAGAAGTTAGGAGATAGTAACTTCCATGACAGTTTCATAGTCTTTTACAGTAAGAAAGCCTCACAACCATCTGCTCAAGAGTTGGTTGTAAAATTAGCCAGGCATGGTGGTGCATGTCTATAATCCCAGCTACTTGGGAGGCTGAGGCAGGAGAATTACTTGAACCCGGGAGGTGGAGGTTGCAGTGAGCCAAGATCATGCCACTGCACTCTAGCCTGGGCAACAGAGTGAGACTCCCGTCTCAAACAAAGAAACAAAGAAACAAACAGAGTAGATTGTGTTCACAAAGATGATGGTAGCTGCTGCCACCAAATGTTGCCCACTGAGACATGAGCAACAGTTTGTCTGCACTGAGAACTCAGCTTCTCAGCAACCTTGTGAAGCAGGAACCATGTGCAACAGGGATGCAGTTCCCATCAATTTTTGTGGATTTGATTCACCTGTTGGAGATGTGCTGCCAAGGAAAGAAAAGTGAAAGTGTGTTTTCAAGACACA<br/>

<br/>
DEBUG: len: 475, AA: MKTVKSTSFLVFFSYIKCQSLRSLLDAAIQYINSQNYPERMETSPTLWLLIFFRKPLMEKYKPLESEISMRSTGPLSHMVTFRMHSPIGTVMKHLEVIHSFIDFLSHVYLLNSCLVMLPRRKKTECFIKLECNKNTLENYLDIIIYVKLFKVFVHVFCPLFDGFFSYIFKFLIDAVIKFNYEVEKAVASLTTPALNRLSYPARSLNPVNSQCHSKEKAYLRKCHTPFIPATSSSTEDQCACKQFTYKQEFLQVFPYEYSKRYPKLAVPVILRVDIVYKNGLTKCYKLENPLECYSCGKEILRRVVCKKQLSCKIIITIYMRSEIVTSMTVSSFTVRKPHNHLLKSWLNPGMVVHVYNPSYLGGGRRITTREVEVAVSQDHATALPGQQSETPVSNKETKKQTEIVFTKMMVAAATKCCPLRHEQQFVCTENSASQQPCEAGTMCNRDAVPINFCGFDSPVGDVLPRKEKKCVFKT<br/>

<br/>
DEBUG: infile: cds_ARG_Human.fa<br/>

<br/>
DEBUG: CDSExonCount: 0, startFrame: ^, stopFrame: <, CDSlength: 85, exonLength: 85<br/>

<br/>
DEBUG: CDSExonCount: 1, startFrame: <, stopFrame: <, CDSlength: 49, exonLength: 51<br/>

<br/>
DEBUG: CDSExonCount: 2, startFrame: <, stopFrame: <, CDSlength: 139, exonLength: 141<br/>

<br/>
DEBUG: CDSExonCount: 3, startFrame: <, stopFrame: <, CDSlength: 130, exonLength: 132<br/>

<br/>
DEBUG: CDSExonCount: 4, startFrame: <, stopFrame: ^, CDSlength: 192, exonLength: 194<br/>

<br/>
DEBUG: CDSExonCount: 5, startFrame: ^, stopFrame: <, CDSlength: 211, exonLength: 211<br/>

<br/>
DEBUG: CDSExonCount: 6, startFrame: <, stopFrame: >, CDSlength: 131, exonLength: 133<br/>

<br/>
DEBUG: CDSExonCount: 7, startFrame: >, stopFrame: ^, CDSlength: 99, exonLength: 100<br/>

<br/>
DEBUG: CDSExonCount: 8, startFrame: ^, stopFrame: <, CDSlength: 313, exonLength: 313<br/>

<br/>
DEBUG: CDSExonCount: 9, startFrame: <, stopFrame: ^, CDSlength: 132, exonLength: 134<br/>

<br/>
DEBUG: CDSExonCount: 10, startFrame: ^, stopFrame: ^, CDSlength: 33, exonLength: 33<br/>

<br/>
DEBUG: len: 482, AA: MKTAKPTSFLVFFSYIKCQSRRSLLDAAIQYINSQNYPGRMETSPTLWLLSFFRKQLMEKYKPQLESEISMRSTGPLSHMVTFRMHSPIGTVMKPLEVIHSCIDLLSWHVYLLNSCLVMLPRRKEDRMPLDKARMQQKYIRELSLRYHNLCEIILKSACSYLLPTFWGCLFVFSYIFVVSYRCCNIYLSRKSSSQLINYPCSKQAILSSKVTQSCEQPVPFQGEGILTQKMLTHLTFHPCHSAVLQKTNLYANNSLTSQNSYKRFLMNIQKDIQSWQFQFEWIYIKMDESATNKTLWNAIAVGSRFCKEWYVKSNYHVKKSLFTEVRRLPHVHSLLQESPTTICSRVGCKIRQALCMPLIPATWEAEAGELLEPGRWRLQAKIAPLHSSLGNRVRLLSQTNRVDCVHKDDGSCCHQMLPTETATVCLHELGFTTLSRNHVQQGCSSNQFLWIWFTCWRAAKERKVKLCFQDTESRKDQKLTS<br/>

<br/>
DEBUG: infile: cds_ARG_Marmoset.fa<br/>

<br/>
DEBUG: CDSExonCount: 0, startFrame: ^, stopFrame: <, CDSlength: 85, exonLength: 85<br/>

<br/>
DEBUG: CDSExonCount: 1, startFrame: <, stopFrame: <, CDSlength: 49, exonLength: 51<br/>

<br/>
DEBUG: CDSExonCount: 2, startFrame: <, stopFrame: <, CDSlength: 139, exonLength: 141<br/>

<br/>
DEBUG: CDSExonCount: 3, startFrame: <, stopFrame: >, CDSlength: 131, exonLength: 133<br/>

<br/>
DEBUG: CDSExonCount: 4, startFrame: >, stopFrame: <, CDSlength: 187, exonLength: 188<br/>

<br/>
DEBUG: CDSExonCount: 5, startFrame: <, stopFrame: >, CDSlength: 209, exonLength: 211<br/>

<br/>
DEBUG: CDSExonCount: 6, startFrame: >, stopFrame: ^, CDSlength: 132, exonLength: 133<br/>

<br/>
DEBUG: CDSExonCount: 7, startFrame: ^, stopFrame: >, CDSlength: 101, exonLength: 101<br/>

<br/>
DEBUG: CDSExonCount: 8, startFrame: >, stopFrame: ^, CDSlength: 309, exonLength: 310<br/>

<br/>
DEBUG: len: 432, AA: MKTAELTSFLIFFSHIKCQSLRSLLEAAIEYINSQNYPERIDTSPTLWLLSFFRKQLMKKYKPQLESEISVRSTGALGHMVTFRVHSLIGTVMKYLEVIHSCIDSLCHVYLLGKQLFSNVAKKKRRQNASRTNALKYIKELSCDIIIHMKLFKVSIHVFCPLFDKVVCFFLYICLSVLILLNLIMKWKKQLAYELPLLTDYPIQDHSIMTARAIPRGRYINSESDHTPDFSSLPLNSSTEDQFVCKQFTYKDFLQEIPHEYSKRYPKLAIPVVLRVDIVYKSLLRKCYKLENPLECHSYGKKIFQRVVVCKSNYYVKNHYDLHEKSGDGNFHDGFIVLYNKKAPQSSAQELVVKLGMVVHTCNPSYLGAGRRIITREMEVAVSDCTTAYQPVRLCLKTGKQTKRVVFTKIMVAAATKCCPLRDEQHTVCTEN<br/>

Checking alignment of cds_ARG_Mouse...<br/>
<br/>
DEBUG: Note: Supplied aligned is a DNA alignment<br/>

Checking alignment of cds_ARG_Chimp...<br/>
<br/>
DEBUG: Note: Supplied aligned is a DNA alignment<br/>

<br/>
DEBUG: Trimming 1 nucleotides from ATGAAGACTGCTAAACCCACATCTTTTCTTGTTTTCTTCAGCTACATTAAATGTCAATCTCGTTAGAGAAGTTTACTAGATGAGGCCATACAGTATATTAATAGCCAGAATTACCCAGAC-AGAATGTAAGAGACATCCCCAACATTATGGTTGCTCAGTTTCTTCAGAAAGCAACTCATGGAGAAGTACAAACCACAGTTAGAAAGTTGAGAGATTTCAATGAGAAGCACAGGACCTTTAAGTCACATGGTTACCTTCAGAATGCACTCACCAATTG--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------GTACAGTTATGAAACATCTAGAAGTCATCCATTCTTGTATAGACTTACTCTCCTGA-CATGTCTACTTGCTATGAAACAGCTGTTTAGTCATGTTGCCAAGAAGAAAAGAAGACAGAATGCCTCTAGATAAAGTTAGAATGCAACA-AAAATATATTAGAGAATTATCTTTGAGATATCATAATTTATGTGAAATTATTTTGAAAAGTGCTTGTTCATATCTTTTGCCCACTTTTTGATGGGGTT----GTTTGTTTTTTCTTATATATTTGTATAAGTTTCTTATAGATGCTGTAATTAAATTTATTTATGAAGTAGAAAAAGCAGT----------------------------------------------------------------------------------------------------------------------------------AGCCAGTTTATAAACTACCCCTGCTCTAAACAGGCTATCTTATCCAGCAAGATCACTCAATCCTGTGAACAGCCAGTGCCATTCCAAGGGGAAGGCATA-TTAACTCAGAAAATGCTGACACACCTGACCTTTCATCCCTGCCACTCAGCAGTTCTACAGAAGACCAGTTTGTATGCAAACAATTCACTTACAAGCCAGAA---TTCCTACAAGCGTTTCCTTATGAATATTCAAAAAGATATCCAAAGTTGGCAGTTCCAGTGATTTTAAGAGTGGATATAGTATATAAAAATGGATTGAGAAAGTGCTACAAATTAGAAAACCCTTTGGAATGCTATAGCTGTGGGAAGCAGATTTTGCAAAGAGTGG---TATGTAAAAAGCAATTATCATGTAAAA-AATCATTATGATTTACATGAGAAGTTAGGAGGTAGTAACTTCCATGACAGGTTCATAGTCTTCTACAGTAAGAAAGCCCCACAACCATCTGCTCAAGAGTTGGTTGTAAAATTTGGCAGGCATTGTGATGCATGCCTCTAATCCCAGCTACTTGGGAGGCTGAGGCAGGAGAATTACTCGAACCTGGGAGGTGGAGGTTGCAGTGAGCCAAGATCGCGCCCCTGCACTCCAGCCTGGGCAACAGGGTG-AGACTCCTGTCTCAAACAA---------------ATAGAGTAGATTGTGTTCACAAAGATGATGGTAGCTGCTGCCACCAAATGTTGCCCACTGAGACATAAGCAACAGTTTGTCTGCATTGAGAACTCG-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------GCTTATGAGCAACCTTGTGAAGCAGGAACCATGTGCAACAG-GGATGCAGTTCCAATCAATTTTTGTGGATTTGGTTCACCTGTTGGAGATGAGCTGCCAAGGAAAGAAAAGTGAAACTGTGTTTTCAAGAGACTGAAAGCAGGAAAGATCAAAAATTGACATCCTAA------------------------------------<br/>

Checking alignment of cds_ARG_Rhesus...<br/>
<br/>
DEBUG: Note: Supplied aligned is a DNA alignment<br/>

<br/>
DEBUG: Trimming 2 nucleotides from ATGAAGACTGTTAAATCCACATCTTTTCTGGTTTTCTTCAGCTACATTAAATGTCAATCTCTTTAGAGAAGTTTATTAGATGCGGCCATACAGTATATTAATAGCCAGAATTACCCAGAG-AGAATGTAAGAGACATCCCCAACATTATGGTTGCTCATCTTCTTCAGAAAGCCACTCATGGAGAAGTACAAACCATAGTTAGAAAGTTGAGAGATTTCAATGAGAAGCACAGGACCTTTAAGTCACATGGTTACCTTCAGAATGCACTCACCAATTG--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------GTACAGTTATGAAACATCTAGAAGTCATCCATTCTTTTATAGACTTCCTCTCCTGA-CATGTCTACTTGCTATGAAACAGCTGTTTAGTCATGTTGCCAAGAAGAAAAA-AGACAGAGTGCTTCTAGATAAAGCTAGAATGCAACA-AAAATACATTAGAGAATTACCTTTGAGATATCATAATTTATGTGAAATTATTTTGAAAAGTGTTTGTTCATGTCTTTTGCCCACTTTTTGATGG-----------CTTTTTTTCTTATATATTT----AAGTTTCTTATAGATGCTGTAATTAAATTTAATTATGAAGTAGAAAAAGCAGT----------------------------------------------------------------------------------------------------------------------------------AGCCAGCTTATGAACTACCCCTGCTCTAAACAGGCTATCCTATCCAGCAAGATCACTCAATCCTGTGAACAGCCAATGCCATTCCAAGGAGAAGGCATA-TTAACTCAGAAAATGCTGACACACCTGACCTTTCATCCCTGCCACT-AGTAGTTCTACAGAAGACCAGTGTGCATGCAAACAATTCACTTACAAGCAAGAA---TTCCTACAAGTGTTTCCTTATGAATATTCAAAAAGATATCCAAAGTTGGCAGTTCCAGTGATTTTAAGAGTGGATATAGTATATAAAAATGGATTGACAAAGTGCTACAAGTTAGAAAACCCTTTGGAATGCTATAGCTGTGGGAAGGAGATTTTGCGAAGAGTGG---TATGTAAAAAGCAATTATCATGTAAAATAATCATTACGATTTACATGAGAAGTTAGGAGATAGTAACTTCCATGACAGTTTCATAGTCTTTTACAGTAAGAAAGCCTCACAACCATCTGCTCAAGAGTTGGTTGTAAAATTAGCCAGGCATGGTGGTGCATGTCTATAATCCCAGCTACTTGGGAGGCTGAGGCAGGAGAATTACTTGAACCCGGGAGGTGGAGGTTGCAGTGAGCCAAGATCATGCCACTGCACTCTAGCCTGGGCAACAGAGTGAGACTCCCGTCTCAAACAAAGAAACAAAGAAACAAACAGAGTAGATTGTGTTCACAAAGATGATGGTAGCTGCTGCCACCAAATGTTGCCCACTGAGACATGAGCAACAGTTTGTCTGCACTGAGAACTCA-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------GCTTCTCAGCAACCTTGTGAAGCAGGAACCATGTGCAACAG-GGATGCAGTTCCCATCAATTTTTGTGGATTTGATTCACCTGTTGGAGATGTGCTGCCAAGGAAAGAAAAGTGAAAGTGTGTTTTCAAGACACA---------------------------------------------------------------------<br/>

Checking alignment of cds_ARG_Human...<br/>
<br/>
DEBUG: Note: Supplied aligned is a DNA alignment<br/>

Checking alignment of cds_ARG_Marmoset...<br/>
<br/>
DEBUG: Note: Supplied aligned is a DNA alignment<br/>

<br/>
DEBUG: name: cds_ARG_Mouse<br/>

<br/>
DEBUG: frame_len: 28<br/>

<br/>
DEBUG: AA_aligned: ATGGAGGTTCTTAAAGCTCTACCCTTCATTGTTTTCTTCAGCTACAGTAAATGTGAGTCTCTCCAAACAAGTTTACTACATGCAGCCAGGCAGCATGTCAGTGGGCAGGATCACCTGGAGGAGAACTTAGGAGTCAGTGTCAGCATTATGGTGGCTCAGTTTCTTCAGGATGCAGCTTATGACCAAGTCCAAACCATAGCTAAAGAGCTGCTGAAGCTTGCTGAAAAGTGTAAGCGTCTCAAACCTGA--GTCACCCTCAGAGTGTGCTCATCAACTGGTGGCTGCCTTCCTTATACACATTTGCAACAACCAAGGCCTGGTGGATAGTCACGTGTTCACTGACTGTTGTAAGATGAAAACTGCAGCAAGACTCAGGTGCTTCCTCTCCTACAAGAAAGACGACGCAGACAGCCATGATACACCCCTGATTCCAAGGCCTGAACTGACCTGTGAGGTGGCTGCAGAGAATAATGTGTCCCTGAAGGAGAGGTTCAGCTATGAAATATCTAGAAGACACCCATTCTTATACGGACCCACTATCTTGAGCATGTCTGCTTGCTATGATACAGCTGTTCAGTCATGTTGTCAAGAAGAGAATGAAACAGAGTGCCTGCGGACAAAGCTAGAACCCATCAGAAAATACATTAGAGGAATATCTGCAAGACATCATCACTTATGTGAAATT-------------------------------------------------------------------------------------------------GGCACCAAATTCAACGAGAAAGTAGCAAGTGCAGTAGAGTTGATTCTACTCACCAAAAAGCAACCTAAAGCTAATTTTTCTGAAATCGCCAAATTGACCATGGATGTTAAAAATCTGCATGAGACCTGTTGTGAGGGGAACACATTAGTGTGTGCCCTGGGCAGGCGCCAGCTTATGAATTACACCTGCTCTAACCAGGCCATCTTATCCAGCAAGTTTGCTGAGTGCTGTGAACAGCCAGAACCATTCCGCGGGGAGTGCATCGTCAACTCAGAAAATGACGACAAGCCTGACCTTTCACCCCTGCCGCTCACAAGGTTTACAGAAGACTGGTCTGTGTGCCAACAATTCAGCGGCAACCAGGACAGCTTTCTACAGGAGTTTCTTTATGAATACTCAAGAAGACACCTGGAGTTGGCGGTTCCCGTGATCTTAAGAGTCTTTGAAACATACAAACGTGTATTGGAGAAATGTTGCAAGCTAGAAAACTCCTTGGAATGCCACCAGCCTGGGAAAGAGATGTTCCAAAAAGTGA---TACTTGAAA-GCCGTGACCAAGTTAAA-ACTTACTGTGACTTACAGAAGCAGCTGGGCAGTAGCAGCTTCCAGGACAGGCTCACGGTGCTTTATACTAAGAAAGCTCCGCAGCTGTCTGCCCAGGAGCTGGTGGTG----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------TTCGTGAAGAAGATGGCAGCGGTTGCCAGGAGATGTTGCCCATTAAGGGATGAGCTGCAGTCCGCTTGCGTGGAGGACCAGACAAAGCTGATTCTTGGAGAGCTGTGCAGAAGACACGGTGGGAAGCCTGTCAATGCAGGAGTGGGCCACTGCTGCGACGACTCCTATGCCTTCAGAAAGCCGTGCTTTGATGACCTGCGAGTCGATGGCACTTACATCTCTCCACCTTTATCTTGTGACCAAGCCATTAGTCTTAAAGAGGACTTGTGCCAAGCTCAGGACGAGGAACTGCAAATGGAAAAGCAGAAGCTCCTCAGCACCCTGGTGAAGCAGAAGCCCCATGCAGCAGAGGATG-TGTTCCATTCCATTGGCGAGAATTTTGTGCAACTGGTGGAGAAGTGCTGCGGTGCACAGAGGAGAG-ACGTGTGTTTTCAAGAAGAGGGACCTCAACTGATCACGAAGTGCCAGTCCCTCTTAGAGGCCAGCTCATCCCAGAGCGGGTTGATGTAG<br/>

<br/>
DEBUG: exon index 0's alignment starts at index 0<br/>

<!--: spam
Content-Type: text/html

<body bgcolor="#f0f0f8"><font color="#f0f0f8" size="-5"> -->
<body bgcolor="#f0f0f8"><font color="#f0f0f8" size="-5"> --> -->
</font> </font> </font> </script> </object> </blockquote> </pre>
</table> </table> </table> </table> </table> </font> </font> </font><body bgcolor="#f0f0f8">
<table width="100%" cellspacing=0 cellpadding=2 border=0 summary="heading">
<tr bgcolor="#6622aa">
<td valign=bottom>&nbsp;<br>
<font color="#ffffff" face="helvetica, arial">&nbsp;<br><big><big><strong>IndexError</strong></big></big></font></td
><td align=right valign=bottom
><font color="#ffffff" face="helvetica, arial">Python 3.6.3: /Library/Frameworks/Python.framework/Versions/3.6/bin/python3<br>Thu Sep 27 15:35:21 2018</font></td></tr></table>
    
<p>A problem occurred in a Python script.  Here is the sequence of
function calls leading up to the error, in the order they occurred.</p>
<table width="100%" cellspacing=0 cellpadding=0 border=0>
<tr><td bgcolor="#d8bbff"><big>&nbsp;</big><a href="file:///Users/csu/research/geneFamilyVisualizer/github-GeneFamilyVisual/dataParser/geneFamilyVis.cgi">/Users/csu/research/geneFamilyVisualizer/github-GeneFamilyVisual/dataParser/geneFamilyVis.cgi</a> in <strong><module></strong>()</td></tr>
<tr><td><font color="#909090"><tt>&nbsp;&nbsp;<small>&nbsp;&nbsp;672</small>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;sys.stderr&nbsp;=&nbsp;sys.stdout<br>
</tt></font></td></tr>
<tr><td><font color="#909090"><tt>&nbsp;&nbsp;<small>&nbsp;&nbsp;673</small>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;print("Content-Type:&nbsp;text/html\n")<br>
</tt></font></td></tr>
<tr><td><font color="#909090"><tt>&nbsp;&nbsp;<small>&nbsp;&nbsp;674</small>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br>
</tt></font></td></tr>
<tr><td bgcolor="#ffccee"><tt>=&gt;<small>&nbsp;&nbsp;675</small>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;main(&nbsp;form&nbsp;)&nbsp;<br>
</tt></td></tr>
<tr><td><font color="#909090"><tt>&nbsp;&nbsp;<small>&nbsp;&nbsp;676</small>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;sys.exit(0)<br>
</tt></font></td></tr>
<tr><td><small><font color="#909090"><strong>main</strong>&nbsp;= &lt;function main&gt;, <strong>form</strong>&nbsp;= FieldStorage(None, None, [])</font></small></td></tr></table>
<table width="100%" cellspacing=0 cellpadding=0 border=0>
<tr><td bgcolor="#d8bbff"><big>&nbsp;</big><a href="file:///Users/csu/research/geneFamilyVisualizer/github-GeneFamilyVisual/dataParser/geneFamilyVis.cgi">/Users/csu/research/geneFamilyVisualizer/github-GeneFamilyVisual/dataParser/geneFamilyVis.cgi</a> in <strong>main</strong>(form=FieldStorage(None, None, []))</td></tr>
<tr><td><font color="#909090"><tt>&nbsp;&nbsp;<small>&nbsp;&nbsp;644</small>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;afa&nbsp;=&nbsp;args.afa&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;#get&nbsp;the&nbsp;aligned&nbsp;provide,&nbsp;or&nbsp;set&nbsp;it&nbsp;as&nbsp;False&nbsp;if&nbsp;user&nbsp;dont&nbsp;provide&nbsp;the&nbsp;aligned<br>
</tt></font></td></tr>
<tr><td><font color="#909090"><tt>&nbsp;&nbsp;<small>&nbsp;&nbsp;645</small>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br>
</tt></font></td></tr>
<tr><td bgcolor="#ffccee"><tt>=&gt;<small>&nbsp;&nbsp;646</small>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;dataProcess(&nbsp;uniqueDir,&nbsp;afa&nbsp;)<br>
</tt></td></tr>
<tr><td><font color="#909090"><tt>&nbsp;&nbsp;<small>&nbsp;&nbsp;647</small>&nbsp;<br>
</tt></font></td></tr>
<tr><td><font color="#909090"><tt>&nbsp;&nbsp;<small>&nbsp;&nbsp;648</small>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;#&nbsp;initialize&nbsp;redirectStr&nbsp;to&nbsp;display&nbsp;output&nbsp;of&nbsp;createSVGtemp.html&nbsp;if&nbsp;called&nbsp;from&nbsp;a&nbsp;CGI&nbsp;form<br>
</tt></font></td></tr>
<tr><td><small><font color="#909090"><em>global</em> <strong>dataProcess</strong>&nbsp;= &lt;function dataProcess&gt;, <strong>uniqueDir</strong>&nbsp;= 'uploadedFiles/a69993d5dfe7d97b33076052ae1d0c2e', <strong>afa</strong>&nbsp;= 'supplementaryMaterials-alignedOnly.afa'</font></small></td></tr></table>
<table width="100%" cellspacing=0 cellpadding=0 border=0>
<tr><td bgcolor="#d8bbff"><big>&nbsp;</big><a href="file:///Users/csu/research/geneFamilyVisualizer/github-GeneFamilyVisual/dataParser/geneFamilyVis.cgi">/Users/csu/research/geneFamilyVisualizer/github-GeneFamilyVisual/dataParser/geneFamilyVis.cgi</a> in <strong>dataProcess</strong>(path='uploadedFiles/a69993d5dfe7d97b33076052ae1d0c2e', afa='supplementaryMaterials-alignedOnly.afa')</td></tr>
<tr><td><font color="#909090"><tt>&nbsp;&nbsp;<small>&nbsp;&nbsp;538</small>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;elif&nbsp;frame_len&nbsp;is&nbsp;0:&nbsp;&nbsp;#&nbsp;done&nbsp;processing&nbsp;this&nbsp;exon<br>
</tt></font></td></tr>
<tr><td><font color="#909090"><tt>&nbsp;&nbsp;<small>&nbsp;&nbsp;539</small>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;exon&nbsp;+=&nbsp;1<br>
</tt></font></td></tr>
<tr><td bgcolor="#ffccee"><tt>=&gt;<small>&nbsp;&nbsp;540</small>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;gene.Frames[exon].append(&nbsp;count&nbsp;)&nbsp;&nbsp;#&nbsp;record&nbsp;the&nbsp;index&nbsp;of&nbsp;the&nbsp;alignment<br>
</tt></td></tr>
<tr><td><font color="#909090"><tt>&nbsp;&nbsp;<small>&nbsp;&nbsp;541</small>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;frame_len&nbsp;=&nbsp;gene.CDS_lens[exon]&nbsp;//&nbsp;3<br>
</tt></font></td></tr>
<tr><td><font color="#909090"><tt>&nbsp;&nbsp;<small>&nbsp;&nbsp;542</small>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;while&nbsp;frame_len&nbsp;==&nbsp;0:&nbsp;#&nbsp;only&nbsp;UTR&nbsp;<br>
</tt></font></td></tr>
<tr><td><small><font color="#909090"><strong>gene</strong>&nbsp;= &lt;__main__.Gene object&gt;, gene.<strong>Frames</strong>&nbsp;= [[0, '^', '&lt;', 85, 85, 0], [1, '&lt;', '&gt;', 50, 52, 29], [2, '&gt;', '^', 138, 139, 47], [3, '^', '&gt;', 212, 212, 94], [4, '&gt;', '^', 132, 133, 165], [5, '^', '&gt;', 98, 98, 210], [6, '&gt;', '^', 129, 130, 243], [7, '^', '&gt;', 215, 215, 289], [8, '&gt;', '^', 132, 133, 361], [9, '^', '&gt;', 98, 98, 406], [10, '&gt;', '^', 138, 139, 439], [11, '^', '&gt;', 227, 227, 486], [12, '&gt;', '^', 132, 133, 562], [13, '^', '^', 69, 69, 607]], <strong>exon</strong>&nbsp;= 14, ].append <em>undefined</em>, <strong>count</strong>&nbsp;= 631</font></small></td></tr></table><p><strong>IndexError</strong>: list index out of range
<br><tt><small>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</small>&nbsp;</tt>args&nbsp;=
('list index out of range',)
<br><tt><small>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</small>&nbsp;</tt>with_traceback&nbsp;=
&lt;built-in method with_traceback of IndexError object&gt;


<!-- The above is a description of an error in a Python program, formatted
     for a Web browser because the 'cgitb' module was enabled.  In case you
     are not reading this in a Web browser, here is the original traceback:

Traceback (most recent call last):
  File "../../dataParser/geneFamilyVis.cgi", line 675, in &lt;module&gt;
    main( form )
  File "../../dataParser/geneFamilyVis.cgi", line 646, in main
    dataProcess( uniqueDir, afa )
  File "../../dataParser/geneFamilyVis.cgi", line 540, in dataProcess
    gene.Frames[exon].append( count )  # record the index of the alignment
IndexError: list index out of range

-->

